### ISSUE TITLE
expansion of function arguments

### ISSUE THREAD

**Issue by behackl** [pynac/pynac#156]
This is actually not really important and more a question of aesthetics:
```
sage: f(x) = function('f')(x)
sage: f(1 + 2*pi*I*x/log(2))
f(2*I*pi*x/log(2) + 1)
sage: _.canonicalize_radical()
f((2*I*pi*x + log(2))/log(2))
sage: _.expand()
f((2*I*pi*x + log(2))/log(2))
```
AFAIK, there is no easy way to get back to the expanded argument (besides substitution, and that's ugly). Should `expand` do that?

**Comment by rwst** [comment]
SymPy does it and I think there is no argument against.

**Comment by behackl** [comment]
This can be closed as of #157.
